lang/go/go.py

```diff
@@ -0,0 +1,211 @@
+from talon import Context, Module, actions
+
+mod = Module()
+ctx = Context()
+ctx.matches = r"""
+code.language: go
+"""
+
+
+ctx.lists["user.code_type"] = {}

```

Comment:
This line may override the .talon-list and probably should be removed. At least on my end, code_type is empty without this change since they are contexts are equally specific.